src/rsa/rsa.rs

```diff
@@ -102,13 +102,9 @@ fn check_public_modulus_and_exponent(
         return Err(error::Unspecified);
     }
 
-    // Step 2 / Step b. XXX: FIPS 186-4 seems to indicate that the minimum
-    // exponent value is 2**16 + 1, but it isn't clear if this is just for
-    // signing or also for verification. We support exponents of 3 and larger
-    // for compatibility with other commonly-used crypto libraries.
-    //
+    // Step 2 / Step b.
```

Comment:
Please disregard the previous comment, since I'm not merging this yet, as I found a minor issue with the first commit.